plugins/modules/zendesk_ticket.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Luis Valle (levalle232@gmail.com)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: zendesk_ticket
+short_description: Manages tickets in Zendesk
+description:
+  - This module allows you to create and delete tickets in Zendesk.
+  - Authentication is handled by the ZENDESK_API class.
+author: "Luis Valle (@elchico2007)"
+options:
+  host:
+    type: str
+    description: The URL of the Zendesk instance.
+    required: true
+  username:
+    type: str
+    description: The Zendesk account username.
+    required: true
+    aliases: ['user']
+  password:
+    type: str
+    description: The Zendesk account password.
+    required: false
+    aliases: ['pass']
+  token:
+    type: str
+    description: The API token for authentication.
+    required: false
+  body:
+    type: str
+    description: The body of the ticket.
+    default: ''
+  priority:
+    type: str
+    description: The priority of the ticket.
+    choices: ['urgent', 'high', 'normal', 'low']
+    default: normal
+  status:
+    type: str
+    description: The status of the ticket.
+    choices: ['new', 'closed', 'resolved']
+    required: true
+  ticket_id:
+    type: int
+    description: The ID of the ticket to be closed or resolved.
+    required: false
+  subject:
+    type: str
+    description: The subject of the ticket.
+    required: false
+'''
+
+EXAMPLES = '''
+  - name: Create a new ticket
+    community.general.zendesk_ticket:
+      username: 'your_username'
+      token: 'your_api_token'
+      host: 'https://yourcompany.zendesk.com'
+      body: 'This is a sample ticket'
+      priority: 'normal'
+      subject: 'New Ticket'
+      status: 'new'
+  - name: Close a ticket
```

Comment:
I would add newlines before the differen examples:
```suggestion

  - name: Close a ticket
```
I would also remove the two spaces indent that **all** examples have. This is simply wasting horizontal space in the rendered examples.